static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -349,6 +373,21 @@ export const issueEventMap: Record<IssueEventKey, string | null> = {
   'issue_error_banner.proguard_misconfigured.clicked':
     'Proguard Potentially Misconfigured Issue Error Banner Link Clicked',
   'issues_tab.viewed': 'Viewed Issues Tab',
+  'issue_views.switched_views': 'Issue Views: Switched Views',
```

Comment:
not sure how many of these are button clicks but if they are ,  we don't have to add an explicit call to `trackAnalytics` - https://develop.sentry.dev/development/analytics/#button-click-analytics

Reply:
Good call - i there are definitely a couple places where we can replace the track analytics with this